admin/app/views/workarea/admin/content_page_copies/new.html.haml

```diff
@@ -0,0 +1,47 @@
+- @page_title = t('workarea.admin.content_page_copies.new.title')
+
+.view
+  .view__header
+    .view__heading
+      - if @page.present?
+        %h1= t('workarea.admin.content_page_copies.new.preselected_title', page: @page.name)
+        %p= t('workarea.admin.content_page_copies.new.preselected_description')
+      - else
+        %h1= t('workarea.admin.content_page_copies.new.title')
+        %p= t('workarea.admin.content_page_copies.new.description')
+
+  .view__container
+    - if @page_copy.present?
+      - @page_copy.errors.full_messages.each do |message|
+        = render_message 'error', message
+
+    = form_tag content_page_copies_path, method: 'post', data: { unsaved_changes: '', copy_id: '' } do
+      .section
+        .grid.grid--center
+          .grid__cell.grid__cell--33
+            - if @page.present?
+              = hidden_field_tag :source_id, @page.id
+            - else
+              .property.property--required
+                = label_tag 'source_id', t('workarea.admin.content_page_copies.new.fields.source_id'), class: 'property__name'
+                = select_tag 'source_id', options_for_pages(params['source_id']), include_blank: true, required: true, data: { remote_select: { source: content_pages_path(format: :json), options: { placeholder: t('workarea.admin.content_blocks.pages.select_placeholder') } }.to_json }
```

Comment:
What user flow makes this a select? Aren't you passing in what page you want to copy in `source_id`?

Reply:
Create content pages: https://github.com/workarea-commerce/workarea/pull/320/files#diff-f8e36588c06502d96be9e204f0b7fbe2R37